viewer-prototype/package.json

```diff
@@ -32,7 +32,7 @@
     "react-virtualized": "^9.21.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.86.0",
-    "timeline-chart": "next",
+    "timeline-chart": "latest",
```

Comment:
keep next, but update the next package (removing the timeline-chart@next in the yarn.lock and running yarn again).